UPGRADE.md

```diff
@@ -80,16 +80,24 @@ With the release of Kuma 2.6.0, we've made some changes to the implementation of
 
 To ensure a smooth transition to Kuma 2.6.0, carefully review your existing configuration files and make necessary adjustments related to denied request responses and RBAC-related Envoy stats.
 
+### Make SI format valid for bandwidth in MeshFaultInjection policy
```

Comment:
This section was shifted up from below.

Reply:
is SI the international term for it? We're talking about this - https://en.wikipedia.org/wiki/International_System_of_Units right?

Reply:
I'm not sure, actually. Because the orginal code block was not added by me and I don't know what was it.